core/src/main/java/io/questdb/TelemetryConfigLogger.java

```diff
@@ -104,14 +149,13 @@ private void tryAddColumn(SqlCompiler compiler, SqlExecutionContext executionCon
         }
     }
 
-    private TableWriter updateTelemetryConfig(
+    private void updateTelemetryConfig(
             CairoEngine engine,
             SqlCompiler compiler,
             SqlExecutionContextImpl sqlExecutionContext,
             TableToken tableToken
     ) throws SqlException {
-        final TableWriter configWriter = engine.getWriter(tableToken, "telemetryConfig");
-        try {
+        try (TableWriter configWriter = engine.getWriter(tableToken, "telemetryConfig")) {
```

Comment:
nit: database startup is very noisy because of the telemetry. We could add a flag to the execution context, to prevent QueryProgress from logging the query executions